### PR TITLE
Initial sign-in-service client and sts documentation split

### PR DESCRIPTION
**Pull request by rileyanderson**
Start work on making distinction between Sign-in service `Client Auth` and `Service Account Auth`. 
- Adds new folders:
    - [Sign-In Service/Engineering Docs/Authentication Types/Client Auth (User)](https://github.com/department-of-veterans-affairs/va.gov-team/tree/dfd0fa2cccfcf25248d7273eb7c6c18310b2e61c/products/identity/Products/Sign-In%20Service/Engineering%20Docs/Authentication%20Types/Client%20Auth%20(User))
    - [Sign-In Service/Engineering Docs/Authentication Types/Service Account Auth (STS)](https://github.com/department-of-veterans-affairs/va.gov-team/tree/dfd0fa2cccfcf25248d7273eb7c6c18310b2e61c/products/identity/Products/Sign-In%20Service/Engineering%20Docs/Authentication%20Types/Service%20Account%20Auth%20(STS))
- The existing documentation under `Sign-In Service/Engineering Docs` will eventually be removed. I left it until the `Authentication Types` folder is cleaned up a bit more and external links are updated to new documentation. 
- Created a new postman collection for [Service Account Auth](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/f1c9526e8c04de099832441cd1778bba8a66e213/teams/vsp/teams/Identity/Product%20Documentation/Sign%20In%20Service/postman_collections/Service_Account_Auth.postman_collection.json)
   - collection for Client Auth is WIP

### CODE REVIEW

File: products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Client Auth (User)/auth_flows/api_oauth.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+## Sign in Service Public Routes
+
+The Sign in Service routes necessary for an API integration are listed below. Routes that are authenticated require a valid SiS `access_token` passed through Bearer Auth. Refresh token authenticated routes requires a `refresh_token` request parameter.
+
+### GET Routes
+
+#### [Authorize](../endpoints/authorize.md) 
+- initiates a session with SiS and prompts the user to enter credentials
+
+#### [Introspect](../endpoints/introspect.md)
```
Comment (joeniquette) [inline products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Client Auth (User)/auth_flows/api_oauth.md:73]: I think we deleted this route, we should probably remove the docs for it as well

File: products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Client Auth (User)/endpoints/introspect.md
Comment (joeniquette) [inline products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Client Auth (User)/endpoints/introspect.md:1]: I think we can remove this

File: products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Service Account Auth (STS)/postman/postman.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+![postman_sts_auth](img/postman_sts_auth.png)
+
+### 4. Make `token` request
+
+1. Make the `token` request by pressing `Send`
+2. If successful, you will see an `access_token` in the response.
+3. The `sts_access_token` variable will be automatically set to the token in the response
+
+### 5. Make `protected sscoped route` request
```
Comment (joeniquette) [inline products/identity/Products/Sign-In Service/Engineering Docs/Authentication Types/Service Account Auth (STS)/postman/postman.md:63]:
```suggestion
### 5. Make `protected scoped route` request
```

**Review by joeniquette — CHANGES_REQUESTED** [review]
minor change and probably can remove introspect stuff

**Review by bramleyjl — COMMENTED** [review]
Looks good, splitting off STS was definitely overdue. Could you please remove the PKCE information from the STS `endpoints/token` document, STS information starts around line 140.

I'll also check out your STS postman collection and work on integrating it into the main Identity Postman collection that we're supporting - I think it makes more sense to maintain one larger collection than multiple individual ones.

**Pull request merged**